xml/security_pam.xml

```diff
@@ -751,7 +751,7 @@ DISPLAY     DEFAULT=${REMOTEHOST}:0.0  OVERRIDE=${DISPLAY}
      <para>
       Finally, remove the debug option from your setup when you are entirely
       satisfied with the performance of it. The <command>pam-config --delete
-      --ldap-debug</command> command turns off debugging for LDAP
+      --sss-debug</command> command turns off debugging for SSSD
```

Comment:
This is still incorrect, this is for debugging the sss pam module, not SSSD as a whole.

Reply:
Turn off/on sssd >> Add a section on configuring SSSD

Reply:
@Firstyear could you take a look , I think we already have a section on SSSD

Reply:
https://documentation.suse.com/sles/15-SP5/html/SLES-all/cha-security-ldap.html#sec-security-ldap-server-sssd